tests/unit/test_appdirs.py

```diff
@@ -4,63 +4,89 @@
 import sys
```

Comment:
And maybe to rename this file?

Reply:
This tests `pip/_internal/utils/appdirs.py`, so the filename is correct unless we rename that as well (along with a bunch of imports). I feel that should be done in a separate PR instead.